gradle/libs.versions.toml

```diff
@@ -10,6 +10,11 @@ darklaf = "3.0.1"
 autoservice-annotations = { module = "com.google.auto.service:auto-service-annotations", version.ref = "autoservice" }
 autoservice-processor = { module = "dev.zacsweers.autoservice:auto-service-ksp", version.ref = "autoserviceKsp" }
 darklaf-nativeUtils = { module = "com.github.weisj:darklaf-native-utils", version.ref = "darklaf" }
+kotlinx-coroutines-core-jvm = {module ="org.jetbrains.kotlinx:kotlinx-coroutines-core-jvm", version ="1.6.1"}
```

Comment:
This should probably updated to match the kotlin version in `gradle.properties`. Once this is merged I'll work on the properties file the single source of truth for versions.